src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';').map(str => str.trim());
+  const result = {};
+
+  styles.forEach(string => {
+    const property = string.split(':');
+
+    if (property[0] && property[1]) {
```

Comment:
nit: Instead of checking 'property[0]' and 'property[1]', use array destructuring and check for 'key' and 'value' directly. For example, 'const [key, value] = property;'